google-cloud-spanner/src/test/java/com/google/cloud/spanner/TimestampBoundTest.java

```diff
@@ -90,8 +89,12 @@ public void exactStaleness() {
 
   @Test
   public void exactStalenessNegative() {
-    expectedException.expect(IllegalArgumentException.class);
-    TimestampBound.ofExactStaleness(-1, TimeUnit.SECONDS);
+    try {
+      TimestampBound.ofExactStaleness(-1, TimeUnit.SECONDS);
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done